preprocessing/nextclade/tests/test.py

```diff
@@ -0,0 +1,404 @@
+import unittest
+
+from loculus_preprocessing.config import get_config
+from loculus_preprocessing.datatypes import (
+    ProcessedEntry,
+    ProcessedEntryFactory,
+    ProcessingAnnotation,
+    UnprocessedEntryFactory,
+)
+from loculus_preprocessing.prepro import process_all
+
+test_config_file = "tests/test_config.yaml"
+
+test_cases = [
+    {
+        "input": UnprocessedEntryFactory.create_unprocessed_entry(
+            metadata_dict={
+                "submissionId": "missing_required_fields",
+            }
+        ),
```

Comment:
Maybe it would even be better to make it a `dict[str, TestCase]` where the keys are the test case descriptions (instead of "hiding" the description in the submissionId).
```suggestion
test_cases = {
    "missing_required_fields": {
        "input": UnprocessedEntryFactory.create_unprocessed_entry(
            metadata_dict={
                "submissionId": "submissionId",
            }
        ),
```

Reply:
done,  created a TestCase with a name field